Formula/stone.rb

```diff
@@ -15,7 +15,7 @@ class Stone < Formula
   option "with-ssl", "SSL support"
 
   def install
-    target = (build.with? "ssl") ? "macosx-ssl" : "macosx"
+    target = build.with?("ssl") ? "macosx-ssl" : "macosx"
```

Comment:
@DomT4 Agreed but not going to pull bottles for this so avoiding changing any behaviour here. Could you make a PR?